### PR TITLE
Level 3 - Added simple Docker image

### PR DESCRIPTION
**Pull request by kbence**
The final container uses the latest alpine image to make the
image smaller.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+WORKDIR /opt/conndetect
+RUN make
+
+FROM alpine
+
+RUN apk add --update gcompat && \
+    rm -rf /var/cache/apk
+COPY --from=build /opt/conndetect/out/conndetect /usr/bin/conndetect
+
+# ENTRYPOINT ["/usr/bin/conndetect"]
```
Comment (knisbet) [inline Dockerfile]: Nitpick: we tend to discourage committing commented out sections unless we really think they'll get revived later on.

**Review by knisbet — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (kbence) [reply Dockerfile]: oh, I actually intended to include that line, I commented it out for testing, thank you! :D

File: README.md [hunk 2]
```diff
@@ -36,9 +36,16 @@ Or you can also run it directly using `go` (from the root directory):
...
+### Running in Docker
 
-### Level 1
+First, build the docker image:
 
-- [x] Currently the program assumes that all connections are incoming, let's use the listening sockets that we drop now to identify the incoming ones. Note: 0.0.0.0 is a wildcard, we have to only match those by ports.
-- [x] Make it a bit more configurable (low hanging fruit, but it'll be useful for testing)
+```shell
+docker build -t conndetect .
+```
```
Comment (wadells) [inline README.md:45]: nit: I'd probably add this to the Makefile

**Review by wadells — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (kbence) [reply README.md:45]: Good point!

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+FROM golang AS build
+
+COPY . /opt/conndetect
+WORKDIR /opt/conndetect
+RUN make
+
+FROM alpine
+
+RUN apk add --update gcompat && \
+    rm -rf /var/cache/apk
```
Comment (webvictim) [inline Dockerfile:10]: FYI you can use `apk add --no-cache` to avoid having to clean up the `/var/cache/apk` directory.

**Pull request merged**